websites/S/Snipes/presence.ts

```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({
+  clientId: "827620297896230912"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async function () {
+const presenceData = {
+      largeImageKey: "logo"
+    },
+    set_timeElapsed = await presence.getSetting("timeElapsed"),
+    set_showButtons = await presence.getSetting("showButtons"),
+    urlpath = window.location.pathname.split("/");
+
+if (set_timeElapsed) {
+  presenceData.startTimestamp = browsingStamp;
+}
+if(!urlpath[1]) {
+  presenceData.details = "Home";
+} else if (urlpath[1] === "c") {
+  const num = (urlpath[2] === 'men' || urlpath[2] === 'women' || urlpath[2] === 'kids') ? "3" : "2";
+  if(document.querySelector("a.b-refinements-category-link.active")){
+    const category = document.querySelector("a.b-refinements-category-link.active").getAttribute("data-name");
+    presenceData.state = (urlpath[parseInt(num) + 1]) ? category : "";
+  }
+  if(set_showButtons) {
+    presenceData.buttons = [
+      {
+        label: 'Show Category',
+        url: window.location.href
```

Comment:
Please provide a reference to which URL (or part of website) this button will redirect to

Reply:
Also wouldn't `View Category` be more appropriate as a label for this?